format/src/main/scala/org/scalafmt/internal/TreeSyntacticGroup.scala

```diff
@@ -52,7 +52,7 @@ object TreeSyntacticGroup {
     case t: Type.ApplyInfix => g.Type.InfixTyp(t.op.value)
     case _: Type.Function => g.Type.Typ
     case _: Type.Tuple => g.Type.SimpleTyp
-    case _: Type.With => g.Type.WithTyp
+    case _: Type.With => g.Type.InfixTyp("with")
```

Comment:
Why not WithTyp?

Reply:
because I added the test case: `A with (B with C)` but if we would normalize it would be the same as `A with B with C`. I will revert this change.